### PR TITLE
WEBEU3 - JavaScript-II - Ifiok -Udoidiok

### PR DESCRIPTION
**Pull request by ifiokudoidiok**
_Start Project_

### CODE REVIEW

File: assignments/array-methods.js [hunk 1]
```diff
@@ -59,27 +59,51 @@ const runners = [
...
 
+
+
+
 // ==== Challenge 5: Be Creative ====
 // Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique problems using one or many of the array methods listed above.
-
+/*
 // Problem 1
+Let us help the event director by arranging the runners alphabetically, according to their surname
```
Comment (josenriagu) [inline assignments/array-methods.js:100]: where are the solutions to the problems?

**Review by josenriagu — APPROVED** [review]
push the solution to the stretch problems and create a new PR

File: assignments/array-methods.js (hunk 1 above)
Comment (ifiokudoidiok) [reply assignments/array-methods.js:100]: I just pushed the solutions for 2 and 3. I do not thing my solution for 1 is completely functional

Comment (josenriagu) [reply assignments/array-methods.js:100]: Make sure all your solutions are working properly.

**Pull request merged**